css-colors/styles.css

```diff
@@ -0,0 +1,50 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+section {
+  padding: 1.5rem 0;
+  background-color: rgb(225 247 255);
+}
+
+h2 {
+  margin-bottom: 3.5rem;
+  font-size: 2rem;
+  text-align: center;
+  font-weight: 300;
+}
+
+form {
+  margin: 0 auto 2rem;
+  text-align: center;
+}
+
+input,
+button {
+  height: 3rem;
+  border-radius: 0.25rem;
+  font-size: 1.1rem;
+  background-color: rgb(124 207 0);
```

Comment:
Remove this background color, and move it down so that it only applies to the button selector. Trying to type into an input with a green background is definitely hard to read! Especially as someone who is colorblind haha

Reply:
lol how did i not see that!
![354354574](https://user-images.githubusercontent.com/122063836/214665666-e72fbcde-0c7a-451d-b19d-55f5f4206906.PNG)
![354354574](https://user-images.githubusercontent.com/122063836/214665666-e72fbcde-0c7a-451d-b19d-55f5f4206906.PNG)